keyring.php

```diff
@@ -263,6 +260,10 @@ static function debug( $str, $level = KEYRING__DEBUG_NOTICE ) {
 		error_log( "Keyring: $str" );
 	}
 
+	/**
+	 * @deprecated
```

Comment:
Shall we add a reason here? "No longer used internally" or something?

Reply:
Sure thing!